pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1124,7 +1124,11 @@ type BackupSpec struct {
 	// BRConfig is the configs for BR
 	BR *BRConfig `json:"br,omitempty"`
 	// MydumperConfig is the configs for mydumper
+	// Deprecated
```

Comment:
It seems that we don't support `mydumper` anymore. I guess we need to say it explicitly in the comment that this field does not take effect anymore, or simply remove this field.

Reply:
I think we can remove  this field.